packages/boot/test/fixtures/application.ts

```diff
@@ -5,13 +5,14 @@
 
 import {RestApplication} from '@loopback/rest';
 import {ApplicationConfig} from '@loopback/core';
+// tslint:disable-next-line:no-unused-variable
```

Comment:
Why is this tslint config needed? AFAIK, tslint knows when an import is required by `tsc` to emit `.d.ts` files and does not complain about unused variables in such case.

Reply:
Not disabling `tslint` gives me the following errors. Isn't `tslint` about code quality while `tsc` is the compiler (and hence doesn't care about unused imports?

```
ERROR: /Users/taranveer/loopback/loopback-next/packages/boot/test/fixtures/application.ts[8, 26]: 'Class' is declared but its value is never read.
ERROR: /Users/taranveer/loopback/loopback-next/packages/boot/test/fixtures/application.ts[8, 33]: 'Repository' is declared but its value is never read.
ERROR: /Users/taranveer/loopback/loopback-next/packages/boot/test/fixtures/application.ts[10, 20]: 'Booter' is declared but its value is never read.
ERROR: /Users/taranveer/loopback/loopback-next/packages/boot/test/fixtures/application.ts[10, 28]: 'Binding' is declared but its value is never read.
```

Reply:
That's really weird! 

Yes, `tsc` should not care about unused imports, unless you enable `noUnusedLocals`. Unfortunately there is a bug in `noUnusedLocals` implementation which prevents us from using it - see https://github.com/Microsoft/TypeScript/issues/21673

Let's not waste too much time on these tslint comments. If they are needed then they are needed, let's move on.